### ISSUE TITLE
get odometry for auto working

### ISSUE THREAD

**Issue by Josh-Westbrook** [frc6377/crescendo_2024#91]
when running a straight 5 meter path on Pathplanner we were coming up 5 inches short and no matter what we did we were always the same amount off but odometry said we were pretty much perfect. we tried slowing the bot down to incredibly slow speeds but it didn't fix it.

**Comment by Josh-Westbrook** [comment]
- try running from a driverstation
- try running from speaker so we can use vision
- try running phoenix pro config again
- verify ALL measurements

**Comment by riversry** [comment]
- We tested the 3 note auton path this morning with consistent error
- We re-measured the wheel radius which was 2 inches and put the new value  of 1.924 inches into vscode
- We put alphabot on blocks and measured the gear ratio for each individual wheel using phoenix tuner x to see the relative encoder positions on each motor - there were no issues with gear ratios
- We calculated the difference between the 2 inch wheel radius and the 1.924 inch wheel radius and realized that it was the source of our consistent error
- Using the new radius value, we ran numerous tests using a 1 meter, 3 meter, and 5 meter auton path with significantly lower error than we had recorded previously

TLDR: the wheel radius was likely our odometry issue

**Comment by caseyjbrotherton** [comment]
Moving this to done.  More work can be done on this, but the main focus for auton and positioning is now the limelight, and we won't get this any closer until we have the competition drivebase
